crates/router/src/db/refund.rs

```diff
@@ -524,7 +527,10 @@ mod storage {
                         .map_err(|error| report!(errors::StorageError::from(error)))
                 }
                 enums::MerchantStorageScheme::RedisKv => {
-                    let key = format!("mid_{}_pid_{}", this.merchant_id, this.payment_id);
+                    let merchant_id = this.merchant_id.clone();
+                    let payment_id = this.payment_id.clone();
```

Comment:
Seems like a redundant clone, We can just do `&this.merchant_id` below

Reply:
this is moved inside redis_entry which is needed by kv wrapper, couldnt find a way to match redis entry lifetime (since its not a reference)